### ISSUE TITLE
Application to the new scence

### ISSUE THREAD

**Issue by zsicl92** [Project-MONAI/VISTA#69]
Hi, @heyufan1995 

When I train the VISTA3D on the new dataset, I encounter the following issue, could you please help me? Thanks a lot.

<img width="749" height="304" alt="Image" src="https://github.com/user-attachments/assets/d633a500-eecf-4611-a180-0c3e5a741af0" />

<img width="610" height="328" alt="Image" src="https://github.com/user-attachments/assets/6ddb01ea-f26e-41a8-8615-6e3bc292e060" />

部分修改的yml文件如下：

drop_label_prob: 0
drop_point_prob: 1
finetune: {activate: false, exclude_vars: null, pretrained_ckpt_name: null}
loss: {_target_: DiceCELoss, include_background: true, sigmoid: false, smooth_dr: 1.0e-05, smooth_nr: 0, softmax: true, squared_pred: true,
  to_onehot_y: true} # 带背景训练
lr_scheduler: {_target_: monai.optimizers.WarmupCosineSchedule, optimizer: $@optimizer,
  t_total: $@num_epochs+1, warmup_multiplier: 0.1, warmup_steps: 0}
max_backprompt: 0
max_foreprompt: 0
max_point: 0 #3
max_prompt: 0 #null
num_epochs: 300
freeze_epoch: 300
freeze_head: "point" #0 #'auto'

发现在运行的过程中，直接跳过训练

logger.debug(f"Iteration skipped due to None prompts at {ds_name}")

想请教一下，我们在制作数据的过程中，必须包含label_prompt、point、label_prompt_pseudo、point_pseudo中的一个数据么？
还是在做键映射的时候需要特殊修改？如果只有Image和label的时候，该怎么去设置呢？

**Comment by heyufan1995** [comment]
if you want to finetune auto only. Just follow https://github.com/Project-MONAI/VISTA/blob/main/vista3d/configs/finetune/train_finetune_word.yaml

Yes. label_prompt、point、label_prompt_pseudo、point_pseudo must exist at least one, otherwise there is no label to train on. 

In your setting, if you want to train auto only, 
```
drop_label_prob: 0
drop_point_prob: 1
finetune: {activate: false, exclude_vars: null, pretrained_ckpt_name: null}
loss: {target: DiceCELoss, include_background: true, sigmoid: false, smooth_dr: 1.0e-05, smooth_nr: 0, softmax: true, squared_pred: true,
to_onehot_y: true} # 带背景训练
lr_scheduler: {target: monai.optimizers.WarmupCosineSchedule, optimizer: $@optimizer,
t_total: $@num_epochs+1, warmup_multiplier: 0.1, warmup_steps: 0}
max_backprompt: null 
max_foreprompt: null # you cannot set to 0 because this defines how many foreground class are sampled
max_point: 0 
max_prompt: null #null
num_epochs: 300
freeze_epoch: 300 # If you want to freeze the encoder and the point branch, you can set it to 300. Otherwise set to 0. 
freeze_head: "point"
```

**Comment by Pengbusi** [comment]
@heyufan1995 Hello，Dear He:
I trained according to train_finetunew_ord.yaml and the YAML configuration you mentioned on Git, and the training loss value remained around 1.0. May I ask if this is a normal phenomenon?
<img width="550" height="771" alt="Image" src="https://github.com/user-attachments/assets/5bae6aaa-d178-4c34-a047-4157166c9d36" />
